hr-data-generator/src/main/java/uk/gov/gchq/palisade/example/hrdatagenerator/types/Manager.java

```diff
@@ -55,42 +55,28 @@ public static Manager generate(final Random random, final String managerType) {
         return manager;
     }
 
-    @Generated
```

Comment:
removing the @Generated annotations?

Reply:
Bad merge

Reply:
still several annotations and methods been removed by the bad merge that need re-adding